change/@ni-nimble-angular-27a0c8b0-01fc-46da-8214-50ceafda6e4b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It's so satisfying to be able to keyboard navigate to the link! Just double checking: do you plan to adopt this in the SLE nav pane once it goes in?

Reply:
Yes